composer.json

```
@@ -4,7 +4,10 @@
     "license": "MIT",
     "description": "XML import/export for Neos, suited for Trados translations",
     "require": {
-        "neos/neos": "~3.0 || ^4.0 || ^5.0"
+        "neos/neos": "~3.0 || ^4.0 || ^5.0 || ^7.0",
```

With PHP 7.4 as a minimum, this follows:
```suggestion
        "neos/neos": "^4.3 || ^5.0 || ^7.0",
```